vscode/src/test/suite/performance/open-jdk.test.ts

```diff
@@ -0,0 +1,135 @@
+import * as assert from 'assert';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as myExtension from '../../../extension';
+import { extensions, languages, Uri, window, workspace } from 'vscode';
+import { assertWorkspace, openFile, runShellCommand, waitCommandsReady } from '../../testutils';
+import { OPENJDK_CHECK_FILES_RESOLVES } from '../../constants';
+
+let lastFileSize = 0;
+let count = 0;
+const checkSymbolsResolved = (path: string): boolean => {
+    let d = languages.getDiagnostics(Uri.file(path));
+    const filterErrorsList = d.filter(el => el.severity == 0);
+    console.log("Filtered Errors List Length: " + filterErrorsList.length);
+    return filterErrorsList.length == 0;
```

Comment:
Perhaps the return should check that there are some diagnostics available and then no errors are reported. 
```js
    return d.length > 0 && filterErrorsList.length == 0;
```

For the first file, this would be done by the separate invocation of `checkExtensionLoaded` and so it may not have any impact when there is only a single file in the check list.